test/dynamo/test_modules.py

```diff
@@ -703,6 +704,9 @@ def test_generation_tag(self):
         m3 = deepcopy(m1)
         self.assertEqual(GenerationTracker.get_generation_value(m3), cur_generation)
 
+    # torch._subclasses.fake_tensor.UnsupportedFakeTensorException: meta converter nyi
+    # due to custom subclass (TensorProxy)
+    @unittest.expectedFailure
```

Comment:
I looked into this, and I think we need to properly fix this. The problem is there is some weird stutter where we don't fakeify the tensor until later, when we have hit the `super().__torch_function__` call. Need some sort of strategy here

Reply:
Alright, agreed, I can peel off the failure and look.

Reply:
This is the torch subclass thing we discussed today. Do you want me to wait until we have a plan there, or is it fine to land as is?